external/include/headers/spirv.h

```diff
@@ -50,12 +50,12 @@
 
 typedef unsigned int SpvId;
 
-#define SPV_VERSION 100
-#define SPV_REVISION 1
+#define SPV_VERSION 10000
```

Comment:
Hmm.  Looking at the version word in the SPIR-V spec, it seems this should be hex and occupy the entire 32-bits of the version word.